src/main/java/com/mabl/integration/jenkins/MablRestApiClientImpl.java

```diff
@@ -119,7 +119,7 @@
                 // TODO why isn't this setting the required Basic auth headers? Hardcoded as work around.
                 .setDefaultCredentialsProvider(getApiCredentialsProvider(restApiKey))
                 .setUserAgent(PLUGIN_USER_AGENT) // track calls @ API level
-                .setConnectionTimeToLive(60, TimeUnit.SECONDS) // use keep alive in SSL API connections
```

Comment:
I don't think we need to touch the connection TTL.  This is just how long the HTTP client will keep connections open after the completion of the last request.  It doesn't have any effect on how long the client will wait before timing out an individual request.  What you need to adjust is the read and write timeouts.  I'm not sure how this is done in apache commons httpclient, but here is how we do it with okhttp:

https://github.com/mablhq/api/blob/master/client/src/main/java/com/mabl/api/client/MablApiClient.java#L127-L135

As a side note, I don't know why we chose to implement our own custom mabl API client rather than extending `MablApiClient` from the `api:client` module.

Reply:
Ah never mind.  I see where this is being done now:

```
                .setConnectionRequestTimeout(REQUEST_TIMEOUT_MILLISECONDS)
                .setSocketTimeout(REQUEST_TIMEOUT_MILLISECONDS)
```